style.css

```diff
@@ -0,0 +1,183 @@
+body {
+  margin: 0;
+}
+
+header {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  width: 100%;
+  height: 48px;
+}
+
+nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  width: 100%;
+}
+
+nav p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+  width: 47%;
+}
+
+.menu {
+  width: 40px;
+  border-radius: 8px;
+  border: hidden;
+  background: #fff;
+}
+
+button:hover {
+  background: #6070ff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+}
+
+button:active {
+  background: #2230d2;
+  border: 1px solid #7f8cff;
+}
+
+button:disabled {
+  border: 1px solid #c1c7d0;
+}
+
+.info {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-image: url("Img/header-shapes\ mobile@2x.png");
+  background-size: 100% 100%;
+  width: 100%;
+  height: 604px;
+}
+
+desc {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 114px 24px;
+  gap: 12px;
+  position: absolute;
+  width: 100%;
+  left: calc(50% - 375px / 2);
+  top: 92px;
+}
+
+.desc h2 {
+  width: 327px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+}
+
+.desc p {
+  width: 327px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.enlaces p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.enlaces img {
+  width: 19px;
+}
+
+.menudesk,
+h1 {
+  display: none;
+}
+
+@media (min-width: 768px) {
```

Comment:
- [Optional] Great job adding the desktop design. Note that a milestone is coming where you will add the desktop styling. You can just focus on the mobile design for now and add the desktop styling then. It is encouraged to build your projects one milestone at a time.